tests/components/zha/test_cover.py

```diff
@@ -131,6 +134,62 @@ def zigpy_keen_vent(zigpy_device_mock):
     )
 
 
+WCAttrs = closures.WindowCovering.AttributeDefs
+WCCmds = closures.WindowCovering.ServerCommandDefs
+WCT = closures.WindowCovering.WindowCoveringType
+WCCS = closures.WindowCovering.ConfigStatus
+
+
+async def test_cover_non_tilt_initial_state(
+    hass: HomeAssistant, zha_device_joined_restored, zigpy_cover_device
+) -> None:
+    """Test ZHA cover platform."""
+
+    # load up cover domain
+    cluster = zigpy_cover_device.endpoints.get(1).window_covering
```

Comment:
minor, but inconsistent in the tests, we can probably just use `endpoints[1]` everywhere.